parallel-consumer-core/src/main/java/io/confluent/parallelconsumer/internal/State.java

```diff
@@ -10,6 +10,14 @@
 public enum State {
     unused,
     running,
+    /**
+     * When paused, the system will stop polling for new records from the broker and also stop submitting work that has already
+     * been polled to the processing pool.
+     *  Committing offsets however
+     * Already submitted in flight work however will be finished (i.e. ).
+     * Parallel stream processor handleWork
+     */
+    paused,
```

Comment:
formatting

and see above for discussion

Reply:
Something went wrong with this Javadoc... will fix it